src/main/bindings/rust/wrapper.rs

```diff
@@ -718,6 +718,7 @@ pub struct _Thread {
     _unused: [u8; 0],
 }
 pub type Thread = _Thread;
+pub type atomic_bool = u8;
```

Comment:
Maybe it would be best to avoid exporting these types to Rust, since it doesn't look like bindgen handles the atomic correctly.